### ISSUE TITLE
README is missing Vite details

### ISSUE THREAD

**Issue by mvolkmann** [honojs/honox#38]
I'm trying to get a basic example of file-based routing working. The README provides a basic `vite.config.ts` file, but it doesn't describe the changes needed in `package.json` to use Vite. I tried changing the "dev" script to just run "vite". But when I run that script with `bun dev` I get the following error:

```text
Error: Cannot find module @rollup/rollup-darwin-x64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
```

Doing what it suggests didn't fix the error for me. I'm running in macOS.

**Comment by yusukebe** [comment]
@mvolkmann 

How about running `bun run dev`?

**Comment by mvolkmann** [comment]
That is the same as `bun dev` and gives the same error.

**Comment by yusukebe** [comment]
Did you create the project by using `bun create hono@latest`?

**Comment by mvolkmann** [comment]
Yes, I did. You can see step-by-step what I did in the "File-based Routing" section of my blog page at https://mvolkmann.github.io/blog/topics/#/blog/hono/.

**Comment by yusukebe** [comment]
Ah,  of course you can create a HonoX project from scratch, but I think it is easier to choose `x-basic`.

<img width="673" alt="Screenshot 2024-02-14 at 14 07 40" src="https://github.com/honojs/honox/assets/10682/c871544e-2849-475e-8472-7109abc82418">

**Comment by mvolkmann** [comment]
Is there a reason why my approach is expected to fail?

**Comment by mvolkmann** [comment]
Thank! Now when I start my server I get "import.meta.glob" is not a function. I think that's related to Bun. Have you seen this error?

**Comment by yusukebe** [comment]
`import.meta.glob` is a Vite's special function.
